Source/Shaders/GlobeVS.glsl

```diff
@@ -175,9 +176,18 @@ void main()
 #endif
 
 #ifdef APPLY_MATERIAL
-    vec3 finalNormal = normalMC;
-    vec3 ellipsoidNormal = normalize(position3DWC.xyz);
-    v_slope = abs(dot(ellipsoidNormal, finalNormal));
+    float northPoleZ = czm_getWgs84EllipsoidEC().radii.z;
+    vec3 northPolePositionMC = vec3(0.0, 0.0, northPoleZ);
+    vec3 ellipsoidNormal = normalize(v_positionMC); // TODO: For a sphere this is correct, but not generally for an ellipsoid!
```

Comment:
The comment is fine, but remove the TODO.

Usually if there is something actionable we would write up an issue, but in this case I think it's ok to remove the TODO and not write an issue.

Reply:
okay @lilleyse, i deleted the "TODO" prefix.